ReactAndroid/src/main/java/com/facebook/react/animated/NativeAnimatedModule.java

```diff
@@ -181,6 +182,11 @@ private void enqueueFrameCallback() {
       mAnimatedFrameCallback);
   }
 
+  @VisibleForTesting
+  public void setNodesManager(NativeAnimatedNodesManager nodesManager) {
```

Comment:
Not sure if there is a better way to do this, we have to inject a mock in here but there is no way currently. I tried using `PowerMockito.whenNew` but could not get it to work.